src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -0,0 +1,348 @@
+)
+def decision_environments_table(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    query = _get_query(models.DecisionEnvironment.objects, since, until)
+    return _copy_table("decision_environments", query, full_path)
+
+
+@register(
+    "event_stream_table",
+    "1.0",
+    format="csv",
+    description="Data on event_streams",
+)
+def event_streams_table(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    query = _get_query(models.EventStream.objects, since, until)
+    return _copy_table("event_streams", query, full_path)
+
+
+@register(
+    "project_table",
+    "1.0",
+    format="csv",
+    description="Data on projects",
+)
+def projects_table(since: datetime, full_path: str, until: datetime, **kwargs):
+    query = _get_query(models.Project.objects, since, until)
+    return _copy_table("projects", query, full_path)
+
+
+@register(
+    "rulebook_table",
+    "1.0",
+    format="csv",
+    description="Data on rulebooks",
+)
+def rulebooks_table(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    query = _get_query(models.Rulebook.objects, since, until)
+    return _copy_table("rulebooks", query, full_path)
+
+
+@register(
+    "rulebook_process_table",
+    "1.0",
+    format="csv",
+    description="Data on rulebook_processes",
+)
+def rulebook_processes_table(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    args = {"started_at": True}
+    query = _get_query(models.RulebookProcess.objects, since, until, **args)
+    return _copy_table("rulebook_processes", query, full_path)
+
+
+@register(
+    "organization_table",
+    "1.0",
+    format="csv",
+    description="Data on organizations",
+)
+def organizations_table(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    args = {"created": True}
+    query = _get_query(models.Organization.objects, since, until, **args)
+    return _copy_table("organizations", query, full_path)
+
+
+@register(
+    "team_table",
+    "1.0",
+    format="csv",
+    description="Data on teams",
+)
+def teams_table(since: datetime, full_path: str, until: datetime, **kwargs):
+    args = {"created": True}
+    query = _get_query(models.Team.objects, since, until, **args)
+
+    return _copy_table("teams", query, full_path)
+
+
+def _datetime_format(dt: datetime) -> str:
+    """Convert datetime object to string."""
+    if dt.microsecond == 0:
+        iso_format = dt.strftime("%Y-%m-%d %H:%M:%S%z")
+    else:
+        iso_format = dt.strftime("%Y-%m-%d %H:%M:%S.%f%z")
+
+    return iso_format[:-2] + ":" + iso_format[-2:]
+
+
+def _get_query(
+    objects: Manager, since: datetime, until: datetime, **kwargs
+) -> str:
+    """Construct sql query with datetime params."""
+    if kwargs.get("started_at"):
+        qs = objects.filter(
+            Q(started_at__gt=since, started_at__lte=until)
+            | Q(updated_at__gt=since, updated_at__lte=until)
+        ).order_by("id")
```

Comment:
This might potentially return duplicates because you are using an OR in the query  I think we want to filter by distinct, am I wrong?

Reply:
Added distinct()